### PR TITLE
Add improvements in json ld definition for list item

### PR DESCRIPTION
**Pull request by anita-chouhan**
Reference: [https://github.com/miranj/miranj.in/blob/dev/templates/miranjv2/_schemas/definitions/listItem.twig](https://github.com/miranj/miranj.in/blob/dev/templates/miranjv2/_schemas/definitions/listItem.twig)

### CODE REVIEW

File: templates/_schemas/definitions/listItem.twig [hunk 1]
```diff
@@ -2,8 +2,9 @@
   
   {% set schema = schema ?? craft.schema.ListItem %}
   {% set schema = schema
```
Comment (rungta) [inline templates/_schemas/definitions/listItem.twig:6]: This should also inherit from `Thing` just like ItemList. https://schema.org/ListItem

File: templates/_schemas/definitions/listItem.twig [hunk 2]
```diff
@@ -2,8 +2,9 @@
...
-    .position(props.listPosition)
-    .url(props.url)
+    .position(props.listPosition|default(null))
+    .name(props.title|default(null))
```
Comment (rungta) [inline templates/_schemas/definitions/listItem.twig]: Both `name` and `url` properties should be auto-set by the `Thing` macro.

**Review by rungta — CHANGES_REQUESTED** [review]

File: templates/_schemas/definitions/listItem.twig (hunk 2 above)
Comment (anita-chouhan) [reply templates/_schemas/definitions/listItem.twig]: Done.

File: templates/_schemas/definitions/listItem.twig (hunk 1 above)
Comment (anita-chouhan) [reply templates/_schemas/definitions/listItem.twig:6]: Okay.

Comment (rungta) [reply templates/_schemas/definitions/listItem.twig:6]: Don't you have to call the `thing` macro somewhere? 🤔

**Review by rungta — CHANGES_REQUESTED** [review]

File: templates/_schemas/definitions/listItem.twig (hunk 1 above)
Comment (anita-chouhan) [reply templates/_schemas/definitions/listItem.twig:6]: Sorry, fixed.

**Review by rungta — APPROVED** [review]

**Pull request merged**